scripts/update-docs.sh

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# This is temporary script to update the docs while our CI is disabled due to 
+# security.
+###############################################################################
+
+# head
+MK_DOCS_SITE_URL="https://head.coda.io/packs/build/latest/" make build-mkdocs
+make publish-docs-head FLAGS=--forceUpload
+
+# staging
+MK_DOCS_SITE_URL="https://staging.coda.io/packs/build/latest/" make build-mkdocs
+make publish-docs-staging FLAGS=--forceUpload
+
+# prod
+MK_DOCS_SITE_URL="https://coda.io/packs/build/latest/" make build-mkdocs
+make publish-docs-prod FLAGS=--forceUpload
+
+echo "\nFinishing pushing to all environments!"
```

Comment:
I would echo in between each just to checkpoint progress.

Reply:
sounds good -- there is output but sounds right to make it explicit